packages/core/src/stylable-transformer.ts

```diff
@@ -464,13 +470,21 @@ export class StylableTransformer {
         const { currentAnchor, metaParts, node, originMeta } = context;
         if (node.type === 'class') {
             CSSClass.hooks.transformSelectorNode({
-                context: { meta: originMeta, diagnostics: this.diagnostics },
+                context: {
+                    meta: originMeta,
+                    diagnostics: this.diagnostics,
+                    resolver: this.resolver,
+                },
                 selectorContext: context,
```

Comment:
fair enough, I feel the same, but not sure what to name it either. The main context is "build context" or "program context", but since it is repeated everywhere I prefer to make it just `context`. 

What would you name the other selector/walk context?

Reply:
leaving this like this for the moment. if we have better naming ideas in the future it's just a small internal change.